tests/vmi_lifecycle_test.go

```diff
@@ -573,6 +575,55 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 				tests.WaitForSuccessfulVMIStart(newVMI)
 			})
+
+			It("[test_id:5275]VMIs with Bridge Networking shouldn't fail after the kubelet restarts", func() {
+				if currentConfiguration.NetworkConfiguration == nil {
+					currentConfiguration.NetworkConfiguration = &v1.NetworkConfiguration{}
+				}
+
+				currentConfiguration.NetworkConfiguration.PermitBridgeInterfaceOnPodNetwork = pointer.BoolPtr(true)
+				kv := tests.UpdateKubeVirtConfigValueAndWait(currentConfiguration)
+				currentConfiguration = kv.Spec.Configuration
+
+				By("starting a VMI on a node")
+				vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).To(BeNil(), "Should submit VMI successfully")
+
+				// Start a VirtualMachineInstance with bridged networking
+				nodeName := tests.WaitForSuccessfulVMIStart(vmi)
+
+				By("restarting kubelet")
+				pod := renderPkillAllPod("kubelet")
+				pod.Spec.NodeName = nodeName
+				_, err = virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Create(pod)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("checking if we see the device plugin restart in the logs")
+				virtHandlerPod, err := kubecli.NewVirtHandlerClient(virtClient).Namespace(flags.KubeVirtInstallNamespace).ForNode(nodeName).Pod()
+				Expect(err).ToNot(HaveOccurred(), "Should get virthandler client for node")
+
+				handlerName := virtHandlerPod.GetObjectMeta().GetName()
+				handlerNamespace := virtHandlerPod.GetObjectMeta().GetNamespace()
+				seconds := int64(10)
+				logsQuery := virtClient.CoreV1().Pods(handlerNamespace).GetLogs(handlerName, &k8sv1.PodLogOptions{SinceSeconds: &seconds, Container: "virt-handler"})
+				Eventually(func() string {
+					data, err := logsQuery.DoRaw()
+					Expect(err).ToNot(HaveOccurred(), "Should get logs")
+					return string(data)
```

Comment:
I think the logs might stream through the kubelet, so when you kill the kubelet this might fail. That could explain this error

```
18:48:56:       Unexpected error:
18:48:56:           <*errors.StatusError | 0xc0049fec80>: {
18:48:56:               ErrStatus: {
18:48:56:                   TypeMeta: {Kind: "", APIVersion: ""},
18:48:56:                   ListMeta: {
18:48:56:                       SelfLink: "",
18:48:56:                       ResourceVersion: "",
18:48:56:                       Continue: "",
18:48:56:                       RemainingItemCount: nil,
18:48:56:                   },
18:48:56:                   Status: "Failure",
18:48:56:                   Message: "an error on the server (\"unknown\") has prevented the request from succeeding (get pods virt-handler-hftm8)",
18:48:56:                   Reason: "InternalError",
18:48:56:                   Details: {
18:48:56:                       Name: "virt-handler-hftm8",
18:48:56:                       Group: "",
18:48:56:                       Kind: "pods",
18:48:56:                       UID: "",
18:48:56:                       Causes: [
18:48:56:                           {
18:48:56:                               Type: "UnexpectedServerResponse",
18:48:56:                               Message: "unknown",
18:48:56:                               Field: "",
18:48:56:                           },
18:48:56:                       ],
18:48:56:                       RetryAfterSeconds: 0,
18:48:56:                   },
18:48:56:                   Code: 500,
18:48:56:               },
18:48:56:           }
18:48:56:           an error on the server ("unknown") has prevented the request from succeeding (get pods virt-handler-hftm8)
18:48:56:       occurred
18:48:56: 
18:48:56:       tests/vmi_lifecycle_test.go:611
```